.goreleaser.yml

```diff
@@ -22,12 +22,13 @@ builds:
       - 7
     targets:
       - darwin_amd64
+      - darwin_386
```

Comment:
Do we really want to support 386 macOS binaries, considering Catalina no longer supports 32-bit apps?

Reply:
Yeah one or the other needs to happen because bonsai would fail as is. I can remove it from bonsai.yml.

Reply:
@amdprophet WDYT?